jre/guide/java/java_jre_insecure_prompt_locked/rule.yml

```diff
@@ -0,0 +1,37 @@
+documentation_complete: true
+
+title: 'Disable User Access to Insecure Prompt of Mobile Code Setting'
+
+description: |-
+    To ensure that users cannot change the mobile code setting for insecure
+    prompts, add <tt>deployment.insecure.jres.locked</tt> to
+    <tt>/etc/.java/deployment/deployment.properties</tt>.
+
+rationale: |-
+    Mobile code has the potential to cause damage to information systems within
+    an organization if used maliciously. It can execute without explicit action
+    from, or notification to, a user. Mobile code is defined as software modules
+    obtained from remote systems, transferred across a network, and then
+    downloaded and executed on a local system without explicit installation or
+    execution by the recipient. Requiring Java to enforce prompting the user
+    prior to executing mobile code will strenghten the security posture of the
+    system. As such, ensuring users cannot change the permission settings which
+    control the insecure prompts for mobile code execution contributes to a more
+    consistent security profile.
```

Comment:
```suggestion
    Mobile code has the potential to cause damage to information systems within
    an organization if used maliciously. It can execute without explicit action
    from, or notification to, a user. Requiring Java to enforce prompting the user
    prior to executing mobile code will strengthen the security posture of the
    system. As such, ensuring users cannot change the permission settings which
    control the insecure prompts for mobile code execution contributes to a more
    consistent security profile.
```